onnxruntime/core/mlas/lib/sgemm.cpp

```diff
@@ -252,16 +256,59 @@ Return Value:
     // time.
     //
 
-    while (CountX >= 16) {
+#if defined(MLAS_USE_SVE) || defined(MLAS_NEON_INTRINSICS)
+    if(MLAS_CPUIDINFO::GetCPUIDInfo().HasArmSve())
+    {
+#if defined(MLAS_USE_SVE)   
+        while (CountX >= MLAS_SGEMM_STRIDEN_THREAD_ALIGN) {
 
-        const float* b = B;
-        size_t y = CountY;
+            const float* b = B;
+            size_t y = CountY;
 
-        do {
+            do {
 
+                    SVE_LOAD_STORE(D,b);
+
+                    D += 16;
```

Comment:
Is the mixed usage of the macro MLAS_SGEMM_STRIDEN_THREAD_ALIGN and hard-coded value of 16 intentional ?